client/post-editor/post-editor.jsx

```diff
@@ -380,6 +381,16 @@ export const PostEditor = React.createClass( {
 								onBlur={ this.copySelectedText }
 								onTextEditorChange={ this.onEditorContentChange } />
 						</div>
+
+						{/* todo need some code somewhere -- maybe in here -- so that if user drops a file on an empty area (an area that isn't a dropzone),
```

Comment:
I think this would be awesome to have as a separate PR, so we can keep this one more on-point with the issue at hand. Also it will decouple the features and allow us to merge the false drop update before merging this PR.